motion-of-no-confidence.md

```diff
@@ -0,0 +1,32 @@
+# Motion of no confidence
+
+The steering committee has [responsibilities][], and it's important
+that those responsibilities are enforceable.  For example, in the
+unlikely event that a committee member doesn't [attend 75% of the
```

Comment:
Necessary to say unlikely? Let's stick to the facts in this document as much as possible.